### PR TITLE
update photon-minimal iso

### PR DESCRIPTION
**Pull request by ppadmavilasom**
- new photon-minimal includes sudo
- removed sudo install from kickstart

### CODE REVIEW

**Comment by ppadmavilasom** [comment]
@akutz i didnt see the need for openssh-server in kickstart. checked after removing it. works okay. i thought openssh was always there in minimal

File: images/capi/packer/ova/linux/photon/http/3/ks.json [hunk 1]
```diff
@@ -16,7 +16,6 @@
     ],
     "postinstall": [
         "#!/bin/sh",
-        "tdnf install -y openssh-server sudo",
```
Comment (codenrhoden) [inline images/capi/packer/ova/linux/photon/http/3/ks.json]: Nice. Yeah if `openssh-server` is already there in minimal, this is find. Always happy to remove a step!

File: images/capi/packer/ova/ova-photon-3.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
   "distro_name": "photon",
   "os_display_name": "VMware Photon OS 64-bit",
   "guest_os_type": "Linux",
-  "iso_url": "http://dl.bintray.com/vmware/photon/3.0/Rev2/iso/photon-minimal-3.0-58f9c74.iso",
-  "iso_checksum": "ae28558e57f5d8aefb8b479c9fac7473079156e1",
+  "iso_url": "https://dl.bintray.com/vmware/photon/3.0/Rev2/iso/Update1/photon-minimal-3.0-a0f216d.iso",
```
Comment (figo) [inline images/capi/packer/ova/ova-photon-3.json]: The new minimal iso size is: 3.42GB,  do i missed something? thx

Comment (ppadmavilasom) [reply images/capi/packer/ova/ova-photon-3.json]: 294MB. i might have mislead you with a different link before. sorry if that's the case

**Comment by figo** [comment]
sorry to hold for better understanding of new iso size.
/hold

**Comment by figo** [comment]
/hold cancel
the size is only 280MB with the correct link

File: images/capi/packer/ova/ova-photon-3.json (hunk 2 above)
Comment (figo) [reply images/capi/packer/ova/ova-photon-3.json]: the iso size is 280MB with correct link

**Pull request merged**